modules/features/profile/src/main/java/au/com/shiftyjelly/pocketcasts/profile/winback/WinbackOfferPage.kt

```diff
@@ -26,77 +56,247 @@ internal fun WinbackOfferPage(
-        ) {
-            Text(
-                text = "Claim Offer",
-                color = Color.Black,
+                .padding(horizontal = 32.dp),
+        )
+        Spacer(
+            modifier = Modifier.height(16.dp),
+        )
+        ClaimFreeOffer(
+            onClick = onClaimOffer,
+            modifier = Modifier.fillMaxWidth(),
+        )
+        Divider()
+        AvailablePlans(
+            onClick = onSeeAvailablePlans,
+            modifier = Modifier.fillMaxWidth(),
+        )
+        Divider()
+        HelpAndFeedback(
+            onClick = onSeeHelpAndFeedback,
+            modifier = Modifier.fillMaxWidth(),
+        )
+        Divider()
+        Spacer(
+            modifier = Modifier.weight(1f),
+        )
+        ContinueToCancellation(
+            onClick = onContinueToCancellation,
+            modifier = Modifier.align(Alignment.CenterHorizontally),
+        )
+        Spacer(
+            modifier = Modifier.height(52.dp),
+        )
+    }
+}
+
+@Composable
+private fun WinbackOfferHeader(
+    modifier: Modifier = Modifier,
+) {
+    Text(
+        text = stringResource(LR.string.winback_offer_header),
+        fontWeight = FontWeight.Bold,
+        fontSize = 28.sp,
+        lineHeight = 38.5.sp,
+        color = MaterialTheme.theme.colors.primaryText01,
+        textAlign = TextAlign.Center,
+        modifier = modifier,
+    )
+}
+
+@Composable
+private fun ClaimFreeOffer(
+    onClick: () -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    RowWithIcon(
+        icon = painterResource(IR.drawable.ic_heart_2),
+        modifier = modifier
+            .clickable(onClick = onClick)
+            .semantics(mergeDescendants = true) { role = Role.Button }
+            .padding(horizontal = 16.dp, vertical = 24.dp),
+    ) {
+        Column {
+            TextH40(
+                text = stringResource(LR.string.winback_offer_free_offer_title),
             )
-        }
-        Box(
-            contentAlignment = Alignment.Center,
-            modifier = Modifier
-                .height(64.dp)
-                .fillMaxWidth()
-                .background(Color(0xFFAE8094))
-                .clickable(onClick = onSeeAvailablePlans),
-        ) {
-            Text(
-                text = "See Plans",
-                color = Color.Black,
+            Spacer(
+                modifier = Modifier.height(4.dp),
             )
-        }
-        Box(
-            contentAlignment = Alignment.Center,
-            modifier = Modifier
-                .height(64.dp)
-                .fillMaxWidth()
-                .background(Color(0xFF85506E))
-                .clickable(onClick = onSeeHelpAndFeedback),
-        ) {
-            Text(
-                text = "Help & Feedback",
-                color = Color.White,
+            TextH60(
+                text = stringResource(LR.string.winback_offer_free_offer_description, "\$3.99"),
+                color = MaterialTheme.theme.colors.primaryText02,
```

Comment:
Should we also localize the price? I mean, put the price converted in the country currency

Reply:
This will come from the API once the logic is connected.